test/import_export_regex_test.dart

```diff
@@ -21,8 +21,8 @@ import 'package:dependency_validator/src/utils.dart';
 void main() {
   group('importExportPackageRegex matches correctly for', () {
```

Comment:
I think we should have similar tests for the SCSS regex.

Reply:
Nice catch. These tests aren't actually running in CI. Time to add them.